FirebaseAdmin/FirebaseAdmin.Snippets/FirebaseAuthSnippets.cs

```diff
@@ -688,6 +689,154 @@ internal static async Task GenerateSignInWithEmailLink()
+                .ListSamlProviderConfigsAsync(listOptions)
+                .GetEnumerator();
+
+            while (await enumerator.MoveNext())
+            {
+                SamlProviderConfig saml = enumerator.Current;
+                Console.WriteLine(saml.ProviderId);
+            }
+
+            // [END list_saml_providers]
+        }
+
+        internal static async Task CreateOidcProviderConfig()
+        {
+            // [START create_oidc_provider]
+            var args = new OidcProviderConfigArgs()
+            {
+                DisplayName = "OIDC provider name",
+                Enabled = true,
+                ProviderId = "oidc.myProvider",
+                ClientId = "CLIENT_ID2",
+                Issuer = "https://oidc.com/CLIENT_ID2",
+            };
+            OidcProviderConfig oidc = await FirebaseAuth.DefaultInstance
+                .CreateProviderConfigAsync(args);
+            Console.WriteLine($"Created new OIDC provider: {oidc.ProviderId}");
+            // [END create_oidc_provider]
+        }
+
+        internal static async Task UpdateOidcProviderConfig()
+        {
+            // [START update_oidc_provider]
+            var args = new OidcProviderConfigArgs()
+            {
+                DisplayName = "OIDC provider name",
+                Enabled = true,
+                ProviderId = "oidc.myProvider",
+                ClientId = "CLIENT_ID",
+                Issuer = "https://oidc.com",
+            };
+            OidcProviderConfig oidc = await FirebaseAuth.DefaultInstance
+                .UpdateProviderConfigAsync(args);
+            Console.WriteLine($"Updated OIDC provider: {oidc.ProviderId}");
+            // [END update_oidc_provider]
+        }
+
+        internal static async Task GetOidcProviderConfig()
+        {
+            // [START get_oidc_provider]
+            OidcProviderConfig oidc = await FirebaseAuth.DefaultInstance
+                .GetOidcProviderConfigAsync("oidc.myProvider");
+            Console.WriteLine($"{oidc.DisplayName}: {oidc.Enabled}");
+            // [END get_oidc_provider]
+        }
+
+        internal static async Task DeleteOidcProviderConfig()
+        {
+            // [START delete_oidc_provider]
+            await FirebaseAuth.DefaultInstance.DeleteProviderConfigAsync("oidc.myProvider");
+            // [END delete_oidc_provider]
+        }
+
+        internal static async Task ListOidcProviderConfigs()
+        {
+            // [START list_oidc_providers]
+            var listOptions = new ListProviderConfigsOptions()
+            {
+                PageToken = "nextPageToken",
+            };
+            IAsyncEnumerator<OidcProviderConfig> enumerator = FirebaseAuth.DefaultInstance
+                .ListOidcProviderConfigsAsync(listOptions)
+                .GetEnumerator();
+
+            while (await enumerator.MoveNext())
+            {
+                OidcProviderConfig saml = enumerator.Current;
```

Comment:
Nit: should we call this `OidcProviderConfig oidc = ...`?

Reply:
Good catch. Done.